services/masterdata/src/lib/core/ui/language-selector/language-selector.component.html

```diff
@@ -0,0 +1,9 @@
+<lens-dropdown
```

Comment:
Cant we use the already existing language picker here? We possible need to update the component a bit to make it more generic, but creating a duplicate component should be a last resort

Reply:
i think this should be a task in another story!

Reply:
It's pretty different. Interesting idea, but not for now. Requires more thought.